src/commands/basemaps-github/create-pr.ts

```diff
@@ -0,0 +1,63 @@
+import { boolean, command, flag, oneOf, option, optional, string } from 'cmd-ts';
+import { logger } from '../../log.js';
+import { verbose } from '../common.js';
+import { CliInfo } from '../../cli.info.js';
+import { Category, MakeCogGithub, parseCategory } from './make.cog.github.js';
+import { ConfigLayer, standardizeLayerName } from '@basemaps/config';
+
+export const CommandCreatePRArgs = {
+  verbose,
+  layer: option({
+    type: string,
+    long: 'layer',
+    description: 'Input config layer import into basemaps-config',
+  }),
+  category: option({
+    type: optional(oneOf(Object.values(Category))),
+    long: 'category',
+    description: 'New Imagery Category, like Rural Aerial Photos, Urban Aerial Photos, Satellite Imagery',
```

Comment:
NIT: `[...Object.values(Cateogory)].join(', ')`

Reply:
You are manually putting the Category values into the description?

Reply:
Oh, my bad, I thought you want to change my `type:`. Yeah, good point, we can update description.